CONTRIBUTING.md

```diff
@@ -1,12 +1,78 @@
-## Before you submit a new issue
-
-* Check if there's a simple solution in the
-  [Troubleshooting](https://github.com/npm/npm/wiki/Troubleshooting)
-  wiki.
-* [Search for similar
-  issues](https://github.com/npm/npm/search?q=Similar%20issues&type=Issues).
-* Ensure your new issue conforms to the [Contributing
-  Guidelines](https://github.com/npm/npm/wiki/Contributing-Guidelines).
-
-Participation in this open source project is subject to the [npm Code
-of Conduct](http://www.npmjs.com/policies/conduct).
+# npm CLI Contributor Guide
+
+## Introduction
+
+Welcome to the npm CLI Contributor Guide! This document outlines the npm CLI repository's process for community interaction and contribution. This includes the issue tracker, pull requests, wiki pages, and, to a certain extent, outside communication in the context of the npm CLI. It defines roles, responsibilities, and procedures, and is an entry point for anyone wishing to contribute their time and effort to making npm a better tool for the JavaScript community!
+
+All interactions in the npm repository are covered by the [npm Code of Conduct](https://www.npmjs.com/policies/conduct)
+
+## Roles
+
+There are three main roles for people participating in the npm issue tracker. Each has a specific set of abilities and responsibilities: [Community members](#community-members), [Collaborators](#collaborators), and [npm, Inc employees](#npm-inc-employees). 
+
+Failure to comply with the expected responsibilities of each role, or violating the Code of Conduct will result in punitive action relative to the transgression, ranging from a warning to full removal from the project, at the discretion of npm employees. 
+
+### Community Members
+
+This includes anyone who may show up to the npm/npm repo with issues, PRs, comments etc. They may not have any other involvement with npm. 
+
+#### Abilities
+
+* Open issues and PRs
+* Comment on issues and PRs
+
+#### Responsibilities
+
+* Comment on issues when they have a reference to the answer. 
+* If community members aren't sure they are correct and don't have a reference to the answer, it's best to leave the issue and try another one.  
+* Defer to collaborators and npm employees for answers.
+* Make sure to search for the troubleshooting doc and search on the issue tracker for similar issues before opening a new one.
+* Any users with urgent support needs are welcome to email support@npmjs.com, and our dedicated support team will be happy to help.
+
+PLEASE don't @ collaborators or npm employees on issues. The CLI team is small, and has many outstanding commitments to fulfill. The hierarchy of attention for the CLI team is as such:
```

Comment:
I have seen this expression several times. And, every time I see this, I think that there is a way to scale.
> The CLI team is small, and has many outstanding commitments to fulfil

If it is a problem that the number of committers is small, we can encourage participation and increase the number of that. That way is how some of the other popular repositories/communities are working on this problem. So as my opinion, I felt something was wrong with `team is small` as a problem to be placed for a long time.

Since npm is a platform instead of a single command and also it is a company, I understand that it is not easy to transfer the permissions. However, I personally think that it is possible to make a quick release in bug fix as semver patch level, and it is often the solution to the fundamental issues. Also, contributors could support it when you don't have enough time :)

I would appreciate if you can consider it as next step. For now, I'd like to see what will happen!